pkg/sdk/agent.go

```diff
@@ -105,3 +106,15 @@ func (sdk *agentSDK) Result() ([]byte, error) {
 
 	return response.File, nil
 }
+
+func (sdk *agentSDK) Attestation() ([]byte, error) {
+	request := &agent.AttestationRequest{}
+
+	response, err := sdk.client.Attestation(context.Background(), request)
```

Comment:
pass context to function, ideally only use context.Background() in main.go

Reply:
This is the footprint used in all other functions in sdk.
Might we need another PR to do that?

Reply:
@SammyOina

Reply:
See
https://github.com/ultravioletrs/cocos-ai/blob/7d47ed7a20dc2048b657b96e486323fe10adf0b4/pkg/sdk/agent.go#L100
and
https://github.com/ultravioletrs/cocos-ai/blob/7d47ed7a20dc2048b657b96e486323fe10adf0b4/pkg/sdk/agent.go#L88

and still [here](https://github.com/ultravioletrs/cocos-ai/blob/7d47ed7a20dc2048b657b96e486323fe10adf0b4/pkg/sdk/agent.go#L74)

Reply:
I think this is simple enough to be in this PR since it is just changing the signatures of the functions in the sdk

Reply:
So should I have it for all other skd methods or just this single one?

Reply:
Might as well do them for the others that require context